sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -750,6 +750,17 @@ case class SetNamespaceProperties(
     copy(namespace = newChild)
 }
 
+/**
+ * The logical plan of the ALTER (DATABASE|SCHEMA|NAMESPACE) ... DEFAULT COLLATION command.
+ */
+case class SetNamespaceCollation(
```

Comment:
can we make it a `UnaryRunnableCommand`? Let's follow `UnsetNamespacePropertiesCommand`